src/scripts/main.js

```diff
@@ -1,3 +1,57 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field tbody');
+const tr = table.querySelector('tr');
```

Comment:
Using a static reference to the first `tr` element for cloning can lead to issues if the table structure changes. Consider dynamically selecting the last row when cloning.